inc/listings.php

```diff
@@ -145,7 +158,8 @@ function cptui_listings() {
 								<?php
 								printf(
 									'<a href="%s">%s</a><br/>
-									<a href="%s">%s</a><br/>',
+									<a href="%s">%s</a><br/>
+									<a href="%s" class="cptui-listings-delete-bottom">%s</a><br/>',
```

Comment:
These should really use named parameters: https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/#variables as discussed on the BEE scrum 8/25 https://webdevstudios.atlassian.net/wiki/spaces/WDS/blog/2022/08/25/2733506561/BEE+Scrum+2022-08-25#LET%E2%80%99S-GET-PEDANTIC-(if-there%E2%80%99s-time)%3A